src/pages/Home.js

```diff
@@ -0,0 +1,49 @@
+import React, { useState } from 'react';
+import { useNavigate } from 'react-router-dom';
+import styled from 'styled-components';
+import citiesData from '../redux/citiesData';
+import CityCard from '../components/CityCard';
+import '../styles/Home.css';
+
+function Home() {
+  const [inputSearch, setInputSearch] = useState('');
+  const navigate = useNavigate();
+  const handleSubmit = (e) => {
+    e.preventDefault();
+    navigate(`/search/${inputSearch}`);
+    setInputSearch('');
+  };
+
+  return (
+    <div>
+      <div className="search-input-container flex justify-center w-[100%] my-10">
+        <Search onSubmit={handleSubmit} className="flex justify-around w-[90%]">
+          <input placeholder="Search City" value={inputSearch} onChange={(e) => setInputSearch(e.target.value)} className=" outline-none border-b-2 text-white border-b-white bg-inherit placeholder:text-white  md:mb-0" />
+          <button type="submit" onClick={handleSubmit} className=" md:hidden block hover:bg-sky-700 text-white bg-sky-800 rounded-md py-1 px-3">Search</button>
+        </Search>
+      </div>
+      <div className="city-container grid grid-cols-2 md:grid-cols-5 border border-t-white">
+        {citiesData.map((city) => <CityCard key={city.name} city={city} />)}
+      </div>
+    </div>
+
+  );
+}
```

Comment:
- [ ] As requested by the previous reviewer to follow the design guidelines, Please fix the below design issues.
  - Please make sure to display two items(countries) in a row. Currently, It's displaying only one item.
  - Please make sure to display a small button on each item as on the given design, That will indicate users going to the details page.

### Your design

![image](https://github.com/Abdulwhbhussain/Metrics-Webapp-Microverse/assets/83803414/61fd802d-02e7-41c9-95c0-dfb5ea2873ba)


### Given design

![image](https://github.com/microhass/thee-worth/assets/83803414/33d636b1-9c33-48d5-8123-cc08978d5032)

- [ ] Please fix the below header and make sure it matches the given design.

![image](https://github.com/Abdulwhbhussain/Metrics-Webapp-Microverse/assets/83803414/d95c072f-12a6-4bf8-9ba8-f1b82f378781)